### PR TITLE
README Rework

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
- Removed redundant symlink
- Moved `bugs.txt`, `enhancement.txt` & `INTERNALS.txt` into `docs` directory
- Reworked README
- Added links to credits, for people I was able to find

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+|:--------:|:------------:|
+| *Atari TOS* | `68030+ CPU` \| `TT` \| `Falcon` \| `Accelerated ST / STE` |
+| *Mac OS X* | `PowerPC` \| `i686 / x86_64` \| `10.4+` |
+| *Windows* | `i686 / x86_64` |
+| *Linux* | **\*** |
+| *FreeBSD* | **\*** |
+| *OS/2* | **\*** |
+| *Haiku* | **\*** |
+
+***OpenDUNE*** *is writtin in `ASNI C(C89)`, only requires `SDL` / `SDL2`* <br>
```
Comment (miniupnp) [inline README.md:43]: ANSI

**Comment by miniupnp** [comment]
what bugs me with removing the README.txt file is that people will have trouble to read the README.md with a plain text editor.

**Comment by ElectronicsArchiver** [comment]
Who would do that though?

Someone interested in code will have a  non-plain text editor anyways.

Besides, to aquire the the code / release one 
will  have to visited the repo in the first place.

The github pages is easy to find and the 
link to it is at the top of the md file as well.

**Comment by miniupnp** [comment]
> Who would do that though?
> 
> Someone interested in code will have a non-plain text editor anyways.
> 
> Besides, to aquire the the code / release one will have to visited the repo in the first place.
> 
> The github pages is easy to find and the link to it is at the top of the md file as well.

the README.txt is included in the .zip archives distribution.
It should be also copied in /usr/share/doc etc by packaging systems. So not everybody interested in OpenDUNE can read a .md file easily.

**Comment by ElectronicsArchiver** [comment]
Why not just use a html file in that case?
Considering no system relevant for OpenDune 
won't also have a webbrowser of some kind.

**Comment by ElectronicsArchiver** [comment]
Since I'm gonna be busy for the next 2 weeks,
heres the txt version re-added, `txt` extension
not neccessary, Github will favor `md`, I'd still
recommend the `html` version as a `txt` 
replacement.

**Comment by miniupnp** [comment]
> Why not just use a html file in that case? Considering no system relevant for OpenDune won't also have a webbrowser of some kind.

no system relevant for OpenDUNE ? like the Atari TT030 ? :)
https://en.wikipedia.org/wiki/Atari_TT030

**Comment by ElectronicsArchiver** [comment]
No system utilizing the repository itself, not the software.
Either way, the previous commit added the txt version back,
what is holding back the merge?

**Pull request closed without merging**